### DISCUSSION TITLE
Meeting notes - June 20th, 2023

### DISCUSSION THREAD

**Discussion by Chris-ECE** [UNECE/GSBPM_GAMSO_Revision#14] [General]
Present : Antti Santaharju, Cory Chobanik, Claudia Brunini, Elodie Kranklader, florian.vucko, Helda Mitre, Kevin McCormack, 

**Finalising the Disseminate phase
_Sub-process 7.5. Manage user support_**

- Agreed to rename this subprocess to “Provide user support”.

**Analyse phase
_General comments_**

- When microdata is the output, there are controls in 5.3 and 6.2, so it’s not clear which tasks are performed in each. – It was suggested to include a sentence in 5.7 that checking of aggregates can be a means of validating microdata
- It was agreed not to add a subprocess for preparing time series data.
- It was agreed to add a point to 6.1 saying that the product can be microdata
- Regarding non-traditional visualized outputs, it was decided that no further action is needed, as 7.2 already mentions the many forms that the product can take (graphics, maps, microdata, etc)
- Regarding inclusion of exploratory data analysis, it was suggested to include mention of this within validation, for example within 6.3.

**_6.1_**

- The comment about precision was already addressesd by the decision to add mention of microdata at the start of phase 6.

**_6.2_**

- It was decided not to rename this to macrovalidation, especially as the output might be microdata.

**_6.2 & 6.3** (Explaining ML-based results)_

- It was suggested to add a sentence in either 6.3 or 6.1 about explaining methodology (e.g. of ML).

**Next meeting**
11th July

**Comment by InKyungChoi** [comment]
Regarding "It was suggested to include a sentence in 5.7 that checking of aggregates can be a means of validating microdata" - I was trying to see how to act on this. Now I think it would be better to put this in 6.2 (Validate outputs) rather than 5.7 (Calculate aggregates). I assume we want to calculate the aggregates _to validate_ the microdata - so the main activity is validation, and we are calculating aggregates for the purpose of validation. Under 6.2 (Validate outputs), we also have "performing macro editing" as an example of validation. @flo7894 - would this be okay?
